scss/components/_menu.scss

```diff
@@ -52,6 +52,11 @@ $menu-item-background-hover: $light-gray !default;
 
   li {
     @include disable-mouse-outline;
+
+    @if $global-flexbox {
```

Comment:
> and in 7 there would be no float anyway I expect

Then how would you get your goverment clients :sweat_smile: who uses IE all the time :wink: 
Ohk removing it!